integration-tests/pom.xml

```diff
@@ -0,0 +1,212 @@
+<!-- Copyright 2017, Oracle Corporation. All Rights Reserved. -->
+<!-- This is unreleased proprietary source code of Oracle Corporation -->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+  
+  <parent>
+    <groupId>oracle.kubernetes</groupId>
+    <artifactId>operator-parent</artifactId>
+    <version>1.0</version>
+  </parent>
+
+    <groupId>oracle.kubernetes</groupId>
+    <artifactId>operator-integration-tests</artifactId>
+    <version>1.0</version>
+
+    <description>Oracle Weblogic Server Kubernetes Operator</description>
+    <name>operator-integration-tests</name>
+    <packaging>jar</packaging>
+
+    <url>https://oracle.github.io/weblogic-kubernetes-operator</url>
+    <inceptionYear>2017</inceptionYear>
+    <licenses>
+      <license>
+        <name>The Universal Permissive License (UPL), Version 1.0</name>
+        <url>https://github.com/oracle/weblogic-kubernetes-operator/blob/master/LICENSE</url>
+      </license>
+    </licenses>
+	<dependencies>
+       <!-- test dependencies -->
+        <dependency>
+            <!-- note: need to include this before junit since junit includes an obsolete version of hamcrest -->
+            <groupId>org.hamcrest</groupId>
+            <artifactId>hamcrest-junit</artifactId>
+            <version>2.0.0.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>junit</groupId>
+            <artifactId>junit</artifactId>
+            <version>${junit-version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.core</groupId>
+            <artifactId>jersey-client</artifactId>
+            <version>${jersey-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.media</groupId>
+            <artifactId>jersey-media-json-processing</artifactId>
+            <version>${jersey-version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jersey.inject</groupId>
+            <artifactId>jersey-hk2</artifactId>
+            <version>${jersey-version}</version>
+        </dependency>
+    </dependencies>
+
+
+    <profiles>
+
+        <profile>
+            <id>java-integration-tests</id>
+            <properties>
+                <src-integration-test>${project.basedir}/src/integration-tests/java</src-integration-test>
+                <resource-integration-test>${project.basedir}/src/integration-tests/resources</resource-integration-test>
+                <docker-image-name>container-registry.oracle.com/middleware/weblogic-kubernetes-operator:latest</docker-image-name>
```

Comment:
I don't understand this.  Why would we only test against the latest from OCR?  Shouldn't we test against what we build?

Reply:
I see that below you build to this image name.  It's still kind of confusing though.

Reply:
I changed the image name based on the git branch name, moved docker build to sh script as the image name is dynamic. please see the latest pom.xml and pullimages.sh is renamed to setupenv.sh.